src/components/resourcesResultsPanel.scss

```diff
@@ -1,23 +1,28 @@
 .resources-results {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
   @media only screen and (max-width: 889px) {
     padding-left: var(--spacing-s);
     padding-right: var(--spacing-s);
   }
 }
 
 .resource {
-  margin-top: calc( var(--spacing-s) * 1.5 );
-  padding: calc( var(--spacing-s) * 3) calc( var(--spacing-s) * 2);
+  margin-top: calc(var(--spacing-s) * 1.5);
   background-color: var(--oyster);
+  max-width: 1200px;
+  padding: 16px;
 
   @media only screen and (max-width: 889px) {
     padding: var(--spacing-s);
   }
 }
 
 .resource__columns {
-  display: flex;
-  margin-left: calc( var(--spacing-s) * -4 );
+  display: grid;
+  grid-template-columns: 3fr 2fr;
```

Comment:
Let's make these columns equal width, so `1fr 1fr`. It feels like the left and right padding gets really big on large screens, while on smaller desktop screens (900-1000px) the content gets really squished. Making the columns each half width seems to alleviate this pretty well.